### PR TITLE
⬆️Update Bpmn-Js

### PR DESCRIPTION
**Pull request by Vyperus**
## What did you change?

This branch updates bpmn-js and uses now rollup for bundling instead of grunt.

## How can others test the changes?

- checkout the branch
- `npm run build`
- `npm link`
- switch to bpmn-studio folder
- `rm -rf node_modules/@process-engine/bpmn-js-custom-bundle`
- `npm link @process-engine/bpmn-js-custom-bundle`
- `npm i`
- `au run`

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [x] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.)
- [x] I've merged the `develop` branch into my branch before finishing this PR.
- [x] I've **not added any other changes** than the ones described above.
- [x] I've mentioned all **PRs, which relate to this one**

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -45,8 +45,8 @@ module.exports = function (grunt) {
     },
   });
 
-  grunt.loadNpmTasks('grunt-browserify');
+  //grunt.loadNpmTasks('grunt-browserify');
```
Comment (alexanderkasten) [inline Gruntfile.js]: We need the browserify task to generate the `dist/bpmn-modeler-custom.js`.
Your are currently only creating the CSS file.

File: Gruntfile.js [hunk 2]
```diff
@@ -45,8 +45,8 @@ module.exports = function (grunt) {
...
   grunt.loadNpmTasks('grunt-contrib-less');
 
-  grunt.registerTask('default', ['less', 'browserify:bower']);
+  grunt.registerTask('default', ['less']);
```
Comment (alexanderkasten) [inline Gruntfile.js]: Why removing `'browserify:bower'` ?

Comment (Vyperus) [reply Gruntfile.js]: My mistake, this needs to bundled of course.

File: Gruntfile.js [hunk 3]
```diff
@@ -44,9 +20,7 @@ module.exports = function (grunt) {
       }
     },
   });
-
-  grunt.loadNpmTasks('grunt-browserify');
   grunt.loadNpmTasks('grunt-contrib-less');
 
-  grunt.registerTask('default', ['less', 'browserify:bower']);
```
Comment (Paulomart) [inline Gruntfile.js]: There is a grunt task for `rollup`: https://www.npmjs.com/package/grunt-rollup

Comment (Paulomart) [reply Gruntfile.js]: Why 👎 ?

Comment (alexanderkasten) [reply Gruntfile.js]: does not work

**Pull request merged**